user/tests.py

```diff
@@ -133,3 +135,58 @@ class LogOutTests(TestCase):
     def test_get_success(self):
         self.response = self.client.get(reverse('logout'))
         self.assertRedirects(self.response, '/')
+
+
+class FollowViewTests(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username='testuser', email='test@gmail.com', password= 'ttt019283est')
+        self.client.login(username='test@gmail.com', password='ttt019283est')
+
+    def test_get_success(self):
+        url = reverse('user:profile', kwargs={'pk': self.user.id})
+        self.response = self.client.get(url)
+        self.assertEqual(self.response.status_code, 200)
+    
+    def test_follow_func(self):
```

Comment:
`follow_func`が何を指しているのか不明なのですが、以下のケースを想定している認識で合っていますか？
```suggestion
    def test_post_success(self):
```

Reply:
あっています。修正しました。